ui/component/or-asset-viewer/src/index.ts

```diff
@@ -98,8 +99,12 @@ export class OrAssetViewer extends subscribe(manager)(translate(i18next)(LitElem
 
         },
         panels: {
+            "underlying assets": {
```

Comment:
Just keep this consistent and call it `group`

Reply:
This is one of the things I ran into: changing the title of a widget. Don's says "underlying assets" (however in Dutch), so how would we approach renaming this?

Reply:
Got it now.